prql-compiler/tests/integration/compile.rs

```diff
@@ -1,164 +0,0 @@
-use insta::assert_snapshot;
```

Comment:
How come we lost these test files? Is it that they're redundant given our unit tests?

Reply:
Oh yeah, I was fixing these tests and noticed that they are duplicates with unit tests and reference tests.

There is no reason to have duplicate tests if we have to maintain them over langugae changes